### PR TITLE
Add DevTools

### PR DESCRIPTION
**Pull request by tanertopal**
I added already the comments for this library https://github.com/jballant/webpack-strip-block but did not yet configure it for production. Whats your opinion on this? I am not sure whats the best way to remove redux-devtools in production so not even the code is imported.

### CODE REVIEW

**Comment by knowbody** [comment]
hmm... 

what about having something like:

``` javascript
export function renderDevTools(store) {
  if (__DEV__) {
    const {DevTools, DebugPanel, LogMonitor} = require('redux-devtools/lib/react');
    return (
      <DebugPanel top right bottom>
        <DevTools store={store} monitor={LogMonitor} />
      </DebugPanel>
    );
  }
  return null;
}
```

and then in `Root.js` something like:

``` javascript
return (
  <div>
    <Provider store={store}>
      ...
    </Provider>

    {renderDevTools(store)}
  </div>
);
```

**Comment by tanertopal** [comment]
Sounds good but wouldn't the DevTools resources bundled anyway? I would like to avoid bundling unused code

**Comment by knowbody** [comment]
yeah this would be bundled

File: src/javascript/views/Login/Index.js [hunk 1]
```diff
@@ -49,17 +48,17 @@ class Login extends Component {
                        hintText='email'
                        floatingLabelText='email'
                        defaultValue='john.doe@example.com'
-                       onKeyDown={::this.submit} /><br/>
+                       onKeyDown={this.submit.bind(this)}/><br/>
```
Comment (knowbody) [inline src/javascript/views/Login/Index.js:51]: binding here creates a new function, it is more efficient to do it in the constructor like:

``` javascript
constructor() {
  super();
  this.submit = this.submit.bind(this);
}
```

**Comment by knowbody** [comment]
closing in favour of #12

**Pull request closed without merging**